### PR TITLE
Remove python to js string

### LINKED ISSUE #857: Remove _python_to_js_string
https://github.com/mozilla/OpenWPM/blob/078395c4ad148a60e20a6dbac549a2287fde2cc3/openwpm/js_instrumentation.py#L23-L37
This function creates a string that looks like JSON while being invalid JSON due to the fact that all `object` fields are unquoted.
We decided to do this, so that once we had de-serialized the object in the WebExtension, the `object` fields directly reference the objects that need instrumenting.
However we have now realized, that it's very easy to get confused and think the string is actually valid JSON.
This has led to a number of unfortunate surprises.
The solution is to eval the `object` field in the WebExtension instead.

### PR DESCRIPTION
**Pull request by vringar**
We now pass along a Dict/JS object for as long as possible, only serialize it to valid JSON and eval at the last reasonable location.
Closes #857

### CODE REVIEW

File: openwpm/utilities/platform_utils.py [hunk 1]
```diff
@@ -125,7 +125,7 @@ def get_configuration_string(manager_params, browser_params, versions):
         # Separate out long profile directory strings
         profile_dirs[browser_id] = str(item.pop("seed_tar"))
         archive_dirs[browser_id] = str(item.pop("profile_archive_dir"))
-        js_config[browser_id] = item.pop("js_instrument_settings")
+        js_config[browser_id] = item.pop("cleaned_js_instrument_settings")
```
Comment (vringar) [inline openwpm/utilities/platform_utils.py:128]: Should we pop off both?

Comment (englehardt) [reply openwpm/utilities/platform_utils.py:128]: What do you mean by both?

**Review by englehardt — APPROVED** [review]
This LGTM, though I'm not sure what your question is asking?

File: openwpm/utilities/platform_utils.py (hunk 1 above)
Comment (vringar) [reply openwpm/utilities/platform_utils.py:128]: Now we only pop off the expanded version, however if a user were to supply their own expanded config, that would still ruin our printing.

Comment (englehardt) [reply openwpm/utilities/platform_utils.py:128]: I see. If both are present than it makes sense to remove both from printing.

**Pull request merged**